api/app/controllers/spree/api/v2/platform/cms_pages_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Spree
+  module Api
+    module V2
+      module Platform
+        class CmsPagesController < ResourceController
+          before_action -> { doorkeeper_authorize! :write, :admin }, only: WRITE_ACTIONS << :toggle_visibility
+
+          def toggle_visibility
```

Comment:
@MatthewKennedy can we handle this by a standard update action?

Reply:
This now uses a simple update on the visible attribute.